sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/instrumentation/EventHubsConsumerInstrumentation.java

```diff
@@ -39,7 +42,14 @@ public Context asyncConsume(String spanName, Message message, String partitionId
         Instant enqueuedTime = MessageUtils.getEnqueuedTime(message.getMessageAnnotations().getValue(), ENQUEUED_TIME_UTC_ANNOTATION_NAME_SYMBOL);
         Context child = parent;
         if (tracer.isEnabled() && !isSync) {
-            child = tracer.startSpan(spanName, tracer.setParentAndAttributes(message, enqueuedTime, parent), ProcessKind.PROCESS);
+            StartSpanOptions options = tracer.createStartOption(SpanKind.CONSUMER, EventHubsTracer.OperationName.PROCESS)
+                .setAttribute(MESSAGE_ENQUEUED_TIME_ATTRIBUTE_NAME, enqueuedTime.atOffset(ZoneOffset.UTC).toEpochSecond());
+
+            if (message.getApplicationProperties() != null) {
```

Comment:
Do we need an entry to the EH ChangeLog_bugsfixed? IIUC, this null check fixes a Receiver stop issue when tracing is enabled, and event application properties are null.

Reply:
this problem was introduced in this PR, if (fortunately) never maid it to main, but it's a great suggestion to add other stuff to changelog